lib/helpers/getNParents.js

```diff
@@ -7,9 +7,13 @@ export function getNParents(element, nParents = 5, parents = []) {
     return parents;
   }
 
+  remainingParents = nParents - 1
+  if (element.tagName == 'button' || element.tagName == 'a') {
```

Comment:
I'd double check with the JS folks that tagName is what you want here.  Also, triple equals.  Also also, these might be capitalized?  You'd need some sample output directly from the library to double check.

Reply:
I think tag name is fine but definitely be careful with case and use strict equals like Pevner pointed out.